### PR TITLE
[UIPO-44] Reusable Section Header Component

### PR DESCRIPTION
**Pull request by mirafedas**

### CODE REVIEW

File: src/components/Heading/Heading.spec.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Heading from './Heading';
+
+describe('<Heading />', () => {
+  test('should match snapshot', () => {
+    let wrapper = shallow(
```
Comment (Hollister009) [inline src/components/Heading/Heading.spec.js]: **const** here and please writte such short code in 1 line

**Review by Hollister009 — APPROVED** [review]
Very good, as written by book 👍

File: src/components/Heading/Heading.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import PropTypes from 'prop-types';
+
+import './Heading.scss';
+
+const CN = 'heading';
+
+const Heading = ({ title, position }) => (
+  <div className={`${CN}__wrapper`}>
+    <div className={`${CN}__line`} />
+    <div className={`${CN}__title ${position === 'right' ? 'right' : 'center'}`}>
```
Comment (taras1313) [inline src/components/Heading/Heading.js]: you can create some enum
const posEnum = {right: 'right', center: 'center'}

then use it just like that
```
    <div className={`${CN}__title ${posEnum[position]}`}>
```

File: src/components/Heading/Heading.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      {title}
+    </div>
+  </div>
+);
+
+Heading.propTypes = {
+  title: PropTypes.string,
+  position: PropTypes.string
```
Comment (taras1313) [inline src/components/Heading/Heading.js]: position: oneOf(Object.keys(posEnum))

what do you think?

Comment (mirafedas) [reply src/components/Heading/Heading.js]: great solution, thank you!

**Review by Nazar-bnm — APPROVED** [review]

**Pull request merged**